.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
Editor specific files should not be checked in 😛

Reply:
Oops.